app/views/merchant/items/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1>Edit Item</h1>
+
```

Comment:
this is another good use case for partials, also would like to refactor this into form_for with resources